group_vars/database.yml

```diff
@@ -11,6 +11,6 @@ mariadb_databases:
   - name: keycloak
 mariadb_mysql_users:
   - name: keycloak-user
-    hosts: "{{ (groups['rhsso'] + groups['datagrid']) | flatten | map('regex_replace', '$', '%') | list }}"
+    hosts: "{{ (groups['rhsso'] + groups['datagrid']) | flatten | map('regex_replace', '_[a-z0-9]*$', '%') | list }}"
```

Comment:
do we still require the changed regex?

Reply:
We have fixed in the inventory file. I think it won't be necessary.